mkosi

```diff
@@ -107,6 +107,7 @@ def create_image(args, workspace):
     print_step("Creating partition table...")
 
     f = tempfile.NamedTemporaryFile(dir = os.path.dirname(args.output), prefix='.mkosi-')
+    subprocess.run(["chattr", "+C", f.name], stdout=subprocess.DEVNULL, stderr=subprocess.DEVNULL, check=True)
     f.truncate(image_size(args))
```

Comment:
well, python exposes ioctl() in the "fnctl" module afaik. All you have to do is invoke it on an fd, with the FS_IOC_SETFLAGS arg as first param, plus the bit field to set as second param...